sdk/identity/azure-identity/src/main/java/com/azure/identity/VisualStudioCodeCredentialBuilder.java

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.identity;
+
+/**
+ * Fluent credential builder for instantiating a {@link VisualStudioCodeCredential}.
+ *
+ * @see VisualStudioCodeCredential
+ */
+public class VisualStudioCodeCredentialBuilder extends CredentialBuilderBase<DefaultAzureCredentialBuilder> {
+
+    private String tenantId;
+
+    /**
+     * Sets the tenant ID of the application.
+     *
+     * @param tenantId the tenant ID of the application.
+     * @return An updated instance of this builder with the tenant id set as specified.
+     */
+    @SuppressWarnings("unchecked")
```

Comment:
I guess this whole builder class will be removed as suggested by @schaabs but if it's not removed then this annotation should be removed.